styles/style.css

```diff
@@ -6,3 +6,166 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+li,
+a {
+  color: black;
+  text-decoration: none;
+  font-family: Arial, Helvetica, sans-serif;
+  margin-right: 10px;
+}
+
+.logo {
+  height: 55px;
+  padding: 10px;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  font-family: Arial, Helvetica, sans-serif;
+  background: #fff;
+  height: 8vh;
+  width: 99vw;
+  font-size: 25px;
+  margin: 5px;
+  padding: 0;
+  position: fixed;
+}
+
+.nav-list {
+  list-style: none;
+  display: flex;
+}
+
+.nav-list li {
+  letter-spacing: 1px;
+  margin-left: 32px;
+}
+
+main {
+  background: url(/images/landing.jpg) no-repeat center center;
+  background-size: cover;
+  height: 100vh;
+  color: #fff;
+  display: block;
+  padding: 300px;
+}
+h1 {
+  font-size: 130px;
+  font-weight: 100;
+  padding-top: 150px;
+}
+
+main p {
+  font-size: 30px;
+  font-weight: lighter;
+}
+main {
+  text-align: center;
+}
+#icons {
+  height: 200px;
+}
+
+.features h2 {
+  font-size: 75px;
+  text-align: center;
+  margin-bottom: 0ch;
+}
+.features article {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  align-content: stretch;
+}
+.features hr {
+  height: 7px;
+  width: 670px;
+  background-color: #00b172;
+  border: none;
+}
+
+.articles {
```

Comment:
to make the 3 articles stay in a single row, you can adjust the width of each article card and also the width of the icon image